src/Microsoft.Health.Dicom.Functions/DurableFunctionsCleanup.cs

```diff
@@ -0,0 +1,73 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using EnsureThat;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
```

Comment:
nit: sort these using statements alphabetically with `System` on top. There should be an option in Visual Studio to do this for you under Edit > IntelliSense > Sort Usings